addons/camera/stringtable.xml

```diff
@@ -24,6 +27,7 @@
         <Key ID="STR_ZEN_Camera_DefaultSpeedCoef">
             <English>Default Speed Coefficient</English>
             <German>Standardmäßiger Geschwindigkeitskoeffizient</German>
+            <Polish>Współczynnik prędkości domyślnej </Polish>
```

Comment:
Trailing space
```suggestion
            <Polish>Współczynnik prędkości domyślnej</Polish>
```

Reply:
This one was marked as resolved but is not fixed.